mephisto/abstractions/blueprints/parlai_chat/parlai_chat_blueprint.py

```diff
@@ -306,8 +307,19 @@ def get_frontend_args(self) -> Dict[str, Any]:
                 "frontend_task_opts": shared_state.frontend_task_opts,
             }
         )
+
+        task_name = self.task_run.to_dict()["task_name"]
+        db = LocalMephistoDB()
+        mephisto_data_browser = MephistoDataBrowser(db)
+        tips = mephisto_data_browser.get_metadata_property_from_task_name(
+            task_name=task_name, property_name="tips"
+        )
+        front_end_task_config_with_metadata = super().update_task_config_with_tips(
+            tips, frontend_task_config
+        )
```

Comment:
I think I did do this in the past, but reverted it because there was some concerns about using the DataBrowser and LocalMephistoDB() in the base blueprint class.

Reply:
Ah okay I recall now what I had considered - Ideally we'd be pulling the `tips` from the task folder on launch, which will be part of the hydra `DictConfig` default arguments, rather than from a query by task name. To this end it may be necessary to implement the "storing tips in an accessible file" here. 

The implementation here would instead look something like:
```
tips = self._load_tips_from_dir(self.args.blueprint.tips_directory)
```
and that would be safe in the `super`.

For the tips saving method, I imagine something like:
```
save_dir = unit.get_task_run().args.blueprint.tips_directory
# write tips out to file
```

And then the default value for `tips_directory` should be interpolatable from `${task_dir}/tips`.

Reply:
I am planning on keeping the tips file in the task's folder.

For example, the static_react_task_with_tips task should have its tips in the `examples/static_react_task_with_tips` folder. The run_task.py file seems to have a property called cfg.task_dir that gets the appropriate field. 

Is there anyway to get this cfg.task_dir in the blueprint.py file?

Reply:
`cfg.task_dir` is a configuration value that is only available from a `TaskConfig` used from a run script. Adding the `tips_directory` argument (or similar) to the blueprint and then autofilling with `${task_dir}/...` locks in that value at runtime and would ensure that future loads of the blueprint have the correct directory filled.

Reply:
What do you mean by autofilling? Not exactly sure what that means.